server.js

```diff
@@ -2257,26 +2259,121 @@ cache(function(data, match, sendBadge, request) {
       return;
     }
     try {
-      var unstable = function(ver) {
-        return /-[0-9A-Za-z.-]+(?:\+[0-9A-Za-z.-]+)?$/.test(ver);
-      };
-      var releases = json[userRepo];
-      if (releases.length == 0) {
-        badgeData.text[1] = 'none';
-        badgeData.colorscheme = 'lightgrey';
-        sendBadge(format, badgeData);
-        return;
+      if (info === 'v') {
+        var unstable = function(ver) {
+          return /-[0-9A-Za-z.-]+(?:\+[0-9A-Za-z.-]+)?$/.test(ver);
+        };
+        var releases = json['releases'];
+        if (releases.length === 0) {
+          badgeData.text[1] = 'none';
+          badgeData.colorscheme = 'lightgrey';
+          sendBadge(format, badgeData);
+          return;
+        }
+        var versions = releases.map(function(version) {
+          return version.version;
+        });
+        var version = latestVersion(versions);
+        if (unstable(version)) {
+          badgeData.colorscheme = "yellow";
```

Comment:
Mmh, I'm indecisive on this. Every other badge uses orange to indicate an unstable version, so conformity would be welcome. However, if there is something about the puppetforge community that gives meaning to that yellow color, I'll follow what the puppetforge community wants.

(Note: I failed to notice that difference while reviewing this: https://github.com/badges/shields/pull/171/files)